jina/peapods/runtimes/jinad/api.py

```diff
@@ -9,12 +9,22 @@
 from ....importer import ImportExtensions
 from ....jaml import JAML
 from ....logging import JinaLogger
+from ....jaml.helper import parse_config_source
 
 
 def _add_file_to_list(_file: str, _file_list: Set, logger: 'JinaLogger'):
     if _file and _file.endswith(('yml', 'yaml', 'py')):
-        if Path(_file).is_file():
-            _file_list.add(_file)
+        _, real_file = parse_config_source(_file,
```

Comment:
what do you think about using `_complete_path(path)`  and making it public `complete_path(path)`?
It would be easier to read.

Reply:
not strong opinion

Reply:
it would shrink 9 lines into 1

Reply:
It may be better, but the number of lines are fake